server/services/taskListService.ts

```diff
@@ -295,7 +295,9 @@ export default class TaskListService {
       return paths.ORDER.SUMMARY.replace(':orderId', order.id)
     }
 
-    return availableTasks[currentTaskIndex + 1].path.replace(':orderId', order.id)
+    return availableTasks[currentTaskIndex + 1].path
+      .replace(':orderId', order.id)
+      .replace(':zoneId', `${order.enforcementZoneConditions.length}`)
   }
```

Comment:
I don't think this will work because zoneId has already been replaced on line 228

Reply:
did not see that you had also solved that issue, removed